src/types.ts

```diff
@@ -28,6 +28,12 @@ export interface CallServiceActionConfig extends BaseActionConfig {
     entity_id?: string | [string];
     [key: string]: any;
   };
+  target?: {
```

Comment:
Is there a reason why you don't reference `HassServiceTarget` as a type from "home-assistant-js-websocket"?

Reply:
The reason is that I was not aware of this type 😃